packages/fx-core/src/plugins/resource/function/ops/provision.ts

```diff
@@ -179,7 +179,7 @@ export class FunctionProvision {
     site.identity = {
       type: DefaultProvisionConfigs.siteIdentityTypeUserAssigned,
       userAssignedIdentities: {
-        [identityName]: {},
+        [identityResourceId]: {},
```

Comment:
We simply replace identity name with resource id, does this also work?

Reply:
We change the key name, but the value is always the resource id.